contentreview-impl/tool/src/java/org/sakaiproject/contentreview/servlet/GradingCallbackServlet.java

```diff
@@ -196,6 +208,21 @@ protected void doPostXml(HttpServletRequest request, HttpServletResponse respons
         return;
     }
 	
+	public Map<String,Object> obtainGlobalTurnitinLTITool(String turnitinSite){
```

Comment:
Didn't want to create a "obtainGlobalTurnitinLTITool" method into the CR-api. I guess I could have extracted it to a different place, though.